retrievalmarket/types.go

```diff
@@ -373,7 +373,7 @@ const (
 // client is interested in, as well as specific parameters the client is seeking
 // for the retrieval deal
 type QueryParams struct {
-	//PayloadCID                 cid.Cid   // optional, query if miner has this cid in this piece. some miners may not be able to respond.
+	PieceCID cid.Cid // optional, query if miner has this cid in this piece. some miners may not be able to respond.
```

Comment:
this seemed to be a typo/leftover from the old version when Piece/Payload CID weren't distinct